### DISCUSSION TITLE
Error/exception fallbacks

### DISCUSSION THREAD

**Discussion by nyurik** [johnkerl/miller#1289] [Q&A]
Is there an "fallback in case of  an error or exception" function in Miller?  Something similar to Excel's `iferror` (as oppose to `iserror`).

Usage patterns:

```
iferror( <something that MAY result in an error>,  <value to use in case of an error>)
```

Alternative - exception-like syntax (probably too verbose here?)

```
try:
  <something that crashes>
except:
  do in case of an error
```

P.S. I searched for a while in the docs, but all "error" searches link to `is_error` function, which is not as convenient to use.

**Comment by johnkerl** [comment]
@nyurik really this needs #1106 ... from very long ago (2015), when I was starting Miller, I wanted a single bad data cell to not crash processing entirely. The intention was good, but the execution -- in many places, cell values get the `MT_ERROR` datatype, which is distinct from `MT_INT` or `MT_STRING` etc, and which renders as (only) `(error)`. What that implementation did was to hide any useful information about what that error cause might have been. #1106 is about doing a better job of surfacing that.

Here ... hmm, I think the `iferror` (which is a _splendid_ idea!!) is actually orthogonal to #1106 (which is a big project). And would be quick and easy to code up, and, I think intuitive and easy for users to reason about.
